config/crd/bases/starrocks.com_starrocksclusters.yaml

```diff
@@ -4104,6 +4135,20 @@ spec:
                         description: the config response key in configmap.
                         type: string
                     type: object
+                  configMaps:
+                    description: the reference for configMap which allow users to
+                      mount any files to container.
+                    items:
+                      properties:
+                        mountPath:
+                          description: Path within the container at which the volume
+                            should be mounted.  Must not contain ':'.
+                          type: string
+                        name:
+                          description: This must match the Name of a Volume.
```

Comment:
looks like this description is not accurate, it is the configmap's name, and the mounted volume name is an internal name, is transparent to the user.